tests/test_skeleton.py

```diff
@@ -0,0 +1,159 @@
+# pylint: skip-file
+# flake8: noqa
+
+import os
+from selenium.webdriver.common.keys import Keys
+from pytest_dash.utils import (
+    wait_for_text_to_equal,
+    wait_for_element_by_css_selector
+)
+from .test_common_features import (
+    init_demo_app,
+    template_test_component_single_prop,
+    PROP_TYPES,
+    COMPONENT_REACT_BASE
+)
+
+# define app name once
+APP_NAME = os.path.basename(__file__).replace(
+    'test_', '').replace(
+        '.py', '').replace(
+            '_', '-')
+
+# pass/fail strings
+PASS = 'PASSED'
+FAIL = 'FAILED'
+
+# define any custom strings here (e.g., if you use
+# a particular CSS selector a lot, assign it to a
+# variable)
+
+
+# Demo app tests
+
+@init_demo_app(APP_NAME)
+def test_click_app_name_from_gallery(dash_threaded, selenium):
+    """Test that clicking on the appropriate app goes to the correct URL."""
+    assert selenium.current_url.replace('http://localhost:8050', '').strip('/') == \
+        'dash-bio/{}'.format(APP_NAME)
+
+    
+# below, write tests for initial conditions; they will most likely
+# make use of wait_for_text_to_equal and other similar functions
+
+@init_demo_app(APP_NAME)
+def test_INITIALCONDITION(dash_threaded, selenium):
+    """Test some initial condition here (e.g., that the correct dataset
+    has been loaded)."""
+
+
+# below, write tests for interactions with the app (e.g., selecting a
+# value from a dropdown or filling in some sort of input); they will
+# most likely make use of send_keys
+
+@init_demo_app(APP_NAME)
+def test_CHANGESOMETHING(dash_threaded, selenium):
+    """Test the results of changing something here (e.g., the value
+    of a dropdown)."""
+
+
+# Component tests 
+
+# for React components, we need to define a way to interact with the
+# props directly (instead of through a graph component); to that end,
+# we define a callback method that defines how exactly a prop is
+# updated
+
+# this callback will be used in the simple test app, which consists of
+# the component, a single button, and two inputs
+
+def COMPONENTNAME_test_props_callback(
```

Comment:
```suggestion
def test_COMPONENTNAME_props_callback(
```
I would prefer sticking with the convention that names of testing functions start with `test_` (we would have to update the function name of the volcano plot test).